client/src/components/BlogPreview.js

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { Row, Col, Icon, Typography, Spin } from 'antd';
+import { Trans } from '@lingui/macro';
+import config from '../config';
+import BlogCard from './BlogCard';
+
+const { Title } = Typography;
+
+const { POSTS_URL } = config;
+
+const LIMIT = 4;
+
+export default ({ title, postSlug = null }) => {
+  const [state, setState] = useState({
+    posts: [],
+    requestError: false,
+    loading: true,
+  });
+
+  const loadPosts = useCallback(async (slug) => {
+    try {
+      const res = await fetch(`${POSTS_URL}/?limit=${LIMIT}&ordering=-published`);
+      if (res.ok) {
+        let { results: posts } = await res.json();
+        if (slug) {
+          posts = posts.filter((post) => post.slug !== slug);
+        }
+        if (posts.length > 3) {
```

Comment:
This is ... peculiar, but I'm presuming there is a good reason behind it ...

Reply:
The `BlogPreview` component was initially used only on the home page with `LIMIT = 3` for the API request, but in the new specs at the bottom of a post page 3 articles should be shown as well. 

In order to filter out the current post from the other articles displayed at the bottom and still have 3 articles, I increased the `LIMIT` to 4 and in case the current post was not among the 4 posts remove the last one. I guess I could change the condition to `if (posts.length === LIMIT)` in order to remove the number 3.